backend/package.json

```diff
@@ -8,7 +8,8 @@
     "prestart": "yarn run build",
     "start": "nodemon src/main/App.ts",
     "test": "jest",
-    "lint": "eslint . --ext .ts,.tsx --fix",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lintFix": "eslint . --ext .ts,.tsx --fix",
```

Comment:
@andrewzulaybar I'm not quite too familiar with this area. I've been finding the interaction between lint and prettier a bit weird at times. Sometimes lint makes us use a single quote, but then prettier changes it back to a double quote. I disabled prettier for now, not too sure what to do or what we can do.

Reply:
@ericrkuo Ah yeah I remember running into that issue before too. Prettier is just supposed to automate fixing the linting issues for you, but sometimes I think they have conflicting rules with the lint rules we set. I think a way around this is either to set the Prettier config to align with our lint rules (https://prettier.io/docs/en/configuration.html) or you could remove it like that and just fix the linting issues manually, whatever you think is easiest!

Reply:
Ohh okay let me take a look to see if I can add that in